src/app/containers/TopStories/index.jsx

```diff
@@ -0,0 +1,68 @@
+import React, { useContext } from 'react';
+import { arrayOf, shape } from 'prop-types';
+import SectionLabel from '@bbc/psammead-section-label';
+import styled from 'styled-components';
+import { StoryPromoLi, StoryPromoUl } from '@bbc/psammead-story-promo-list';
+import { GEL_GROUP_3_SCREEN_WIDTH_MIN } from '@bbc/gel-foundations/breakpoints';
+import {
+  GEL_SPACING_DBL,
+  GEL_SPACING_TRPL,
+} from '@bbc/gel-foundations/spacings';
+
+import topStories from '#pages/StoryPage/topStories.json';
+import { storyItem } from '#models/propTypes/storyItem';
+import { ServiceContext } from '#contexts/ServiceContext';
+import { GridItemConstrainedLarge } from '#lib/styledGrid';
```

Comment:
It would be good to use the Grid from the Psammead package `import Grid from '@bbc/psammead-grid';`, since `GridItemConstrainedLarge` will be deprecated very soon.  Usage instructions for psammead-grid are here: https://github.com/bbc/psammead/blob/latest/packages/components/psammead-grid/README.md

Reply:
This issue already tracks that https://github.com/bbc/simorgh/issues/5795